<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Mainwindow</class>
 <widget class="QMainWindow" name="Mainwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1055</width>
    <height>567</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mainwindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>1061</width>
      <height>401</height>
     </rect>
    </property>
    <property name="accessibleName">
     <string/>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Visiteur</string>
     </attribute>
     <widget class="QTableView" name="visiteurTable">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>681</width>
        <height>241</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">selection-color: rgb(0, 255, 255);</string>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderMinimumSectionSize">
       <number>25</number>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>150</number>
      </attribute>
      <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderDefaultSectionSize">
       <number>25</number>
      </attribute>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>290</y>
        <width>581</width>
        <height>41</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="ajoutVisteur">
         <property name="text">
          <string>Ajouter</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="modifVisteurBtn">
         <property name="text">
          <string>Modifier</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="SupVisiteurBtn">
         <property name="text">
          <string>Supprimer</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QFrame" name="frame_3">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>10</y>
        <width>361</width>
        <height>241</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
background-color: rgb(188, 218, 255);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <widget class="QFrame" name="frame_4">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>260</y>
        <width>361</width>
        <height>101</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(188, 218, 255);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Site</string>
     </attribute>
     <widget class="QTableView" name="tableSite">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>731</width>
        <height>221</height>
       </rect>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderMinimumSectionSize">
       <number>25</number>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>150</number>
      </attribute>
      <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderDefaultSectionSize">
       <number>25</number>
      </attribute>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>270</y>
        <width>501</width>
        <height>61</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QPushButton" name="AjoutSite">
         <property name="text">
          <string>Ajouter</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ModifSiteBtn">
         <property name="text">
          <string>Modifier</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="SuppSite">
         <property name="text">
          <string>Supprimer</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QFrame" name="frame_2">
      <property name="geometry">
       <rect>
        <x>740</x>
        <y>260</y>
        <width>321</width>
        <height>121</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(188, 218, 255);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <widget class="QFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>740</x>
        <y>30</y>
        <width>321</width>
        <height>231</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
background-color: rgb(188, 218, 255);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>181</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 700 14pt &quot;Segoe Script&quot;;</string>
       </property>
       <property name="text">
        <string>Info sur le site</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>Visiter</string>
     </attribute>
     <widget class="QTableView" name="tableVisite">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>741</width>
        <height>261</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>-1</x>
        <y>260</y>
        <width>741</width>
        <height>61</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="AjoutVisite">
         <property name="text">
          <string>Ajouter</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ModifVisite">
         <property name="text">
          <string>Modifier</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="SupVisite">
         <property name="text">
          <string>Supprimer</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>10</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Recherche</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="rechNomNum"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1055</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
